README.md

```diff
@@ -75,6 +75,15 @@ Please visit the [Kubernetes GSoC page](https://git.k8s.io/community/mentoring/g
 -	Mentor(s): Andy Zhang (@andyzhangx)
 -	Issue: https://github.com/kubernetes/org/issues/344
 
+### Kubernetes Dashboard
```

Comment:
@maciaszczykm This will go as a project idea for Kubernetes directly, so this isn't needed :)

Reply:
Okay, will change 👍

Reply:
Done, thanks :) Still thinking how to describe the second idea...